apps/emqx_bridge/i18n/emqx_bridge_schema.conf

```diff
@@ -195,7 +195,7 @@ emqx_bridge_schema {
 
     metric_sent_inflight {
```

Comment:
nit: inflight is sent
```suggestion
    metric_inflight {
```

Reply:
Will fix it here: https://github.com/emqx/emqx/pull/9642